tests/unit/computeapi/host/block_allocator.cpp

```diff
@@ -78,7 +78,7 @@ int hpx_main(boost::program_options::variables_map& vm)
 
     std::cout << "using seed: " << seed << std::endl;
     std::mt19937 gen(seed);
-    std::uniform_int_distribution<> dis(0, 511);
+    std::uniform_int_distribution<> dis(1, 512);
```

Comment:
Shouldn't we now leave the original code in place?

Reply:
I'm fine with that as well. I thought it's nicer to have two separate cases, one for `count = 0` and one for `count != 0` so that in case there's a useful stacktrace one can see which case failed, but don't mind changing it back either.

Reply:
Sure, makes sense.